sdks/java/io/cdap/src/main/java/org.apache.beam.sdk.io.cdap/PluginWrapper.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.sdk.io.cdap;
+
+import io.cdap.cdap.api.plugin.PluginConfig;
+
+/**
+ * Class wrapper for a plugin.
+ */
+public class PluginWrapper {
```

Comment:
How about dropping the -Wrapper suffix, because a wrapper is written for a class, but we are describing an entity CDAP plugin, one such class does not exist, because the plugin is represented by several classes in CDAP?

Maybe call it like CdapPlugin. I think that makes it easier to understand what we're working with.